src/Plugin.php

```diff
@@ -64,6 +64,8 @@ public function bootstrap(PluginApplicationInterface $app): void
      */
     public function console(CommandCollection $commands): CommandCollection
     {
-        return $commands->add('worker', WorkerCommand::class);
+        return $commands
+            ->add('queue work', WorkerCommand::class)
```

Comment:
Don't really like the `work` vs `Worker` naming mismatch.

Reply:
Updated to `queue worker`.